includes/class-wcs-object-data-cache-manager.php

```diff
@@ -150,7 +146,18 @@ public function prepare_object_changes( $object, $generate_type = 'changes_only'
 					];
 				}
 
-				break;
```

Comment:
This `break` wasn't doing anything before because it was at the very end of `foreach` anyway. In this PR, I've changed it to a `break 2` so that if we find a metakey in the `$meta_data` than it hasn't been deleted and the later code outside this `foreach` can assume that it must have been deleted.